enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1601,44 +1601,82 @@ func (p *PebbleCache) FindMissing(ctx context.Context, resources []*rspb.Resourc
 	}
 	defer db.Close()
 
+	lockedResources, err := p.lockResourceKeys(ctx, resources)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I don't recall but do we have data on how much memory/cpu we save by creating a single iterator per FindMissing call? We could avoid this complexity if we didn't need to have a single iterator for the lifetime of the FindMissing call.

Reply:
Could we use https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/util/pebble/pebble.go#L575 instead?

Reply:
See this PR: https://github.com/buildbuddy-io/buildbuddy/pull/5877